fontforge/python.c

```diff
@@ -14066,6 +14066,150 @@ return (NULL);
 return( Py_BuildValue("d",val));
 }
 
+static PyObject *PyFF_Font_get_style_set_names(PyFF_Font *self, void *UNUSED(closure)) {
+    int cnt, i;
+    SplineFont *sf = self->fv->sf;
+    struct otffeatname *fn;
+    struct otfname *on;
+    PyObject *ss_names_tuple, *ss_name_spec;
+    char *lang_str, tag_str[5];
+
+    if ( CheckIfFontClosed(self) )
```

Comment:
If we need to check this, then presumably the `self->fv->sf` access above isn't safe.